doc/onboarding-extras.md

```diff
@@ -5,40 +5,40 @@
+| `lib/repl`                               | @nodejs/repl                                                          |
 | `lib/{_}stream{*}`                       | @nodejs/streams                                                       |
-| `lib/timers`                             | @fishrock123, @misterdjules                                           |
-| `lib/util`                               | @bnoordhuis, @cjihrig, @evanlucas                                     |
-| `lib/zlib`                               | @addaleax, @bnoordhuis, @indutny                                      |
+| `lib/timers`                             | @nodejs/timers                                                        |
+| `lib/util`                               | @nodejs/util                                                          |
+| `lib/zlib`                               | @nodejs/zlib                                                          |
 | `src/async-wrap.*`                       | @nodejs/async\_hooks                                                  |
 | `src/node_api.*`                         | @nodejs/n-api                                                         |
 | `src/node_crypto.*`                      | @nodejs/crypto                                                        |
 | `test/*`                                 | @nodejs/testing                                                       |
-| `tools/node_modules/eslint`, `.eslintrc` | @not-an-aardvark, @silverwind, @trott                                 |
+| `tools/node_modules/eslint`, `.eslintrc` | @nodejs/linting                                                       |
 | build                                    | @nodejs/build                                                         |
 | `src/module_wrap.*`, `lib/internal/loader/*`, `lib/internal/vm/Module.js` | @nodejs/modules                      |
 | GYP                                      | @nodejs/gyp                                                           |
 | performance                              | @nodejs/performance                                                   |
 | platform specific                        | @nodejs/platform-{aix,arm,freebsd,macos,ppc,smartos,s390,windows}     |
 | python code                              | @nodejs/python                                                        |
-| upgrading c-ares                         | @jbergstroem                                                          |
-| upgrading http-parser                    | @jbergstroem, @nodejs/http                                            |
-| upgrading libuv                          | @saghul                                                               |
+| upgrading c-ares                         | @rvagg,                                                               |
+| upgrading http-parser                    | @nodejs/http nodejs/http2                                             |
```

Comment:
Nit: missing comma and `@`  (`@nodejs/http nodejs/http2` -> `@nodejs/http, @nodejs/http2`)?